framework/src/utils/FVUtils.C

```diff
@@ -0,0 +1,32 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "MooseVariableFV.h"
+#include "FVUtils.h"
+
+namespace Moose
+{
+namespace FV
+{
+
+bool
+elemHasFaceInfo(const Elem & elem, const Elem * const neighbor)
```

Comment:
is there any way you could use this function in MooseMesh when creating the face infos?
It s the same logic